x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -688,48 +688,24 @@ export function registerEncryptedSavedObjects(
       { key: 'ssl', dangerouslyExposeValue: true },
       { key: 'password', dangerouslyExposeValue: true },
     ]),
-    attributesToExcludeFromAAD: new Set([
-      'output_id',
-      'name',
-      'type',
-      'is_default',
-      'is_default_monitoring',
-      'hosts',
-      'ca_sha256',
-      'ca_trusted_fingerprint',
-      'config',
-      'config_yaml',
-      'is_preconfigured',
-      'proxy_id',
-      'version',
-      'key',
-      'compression',
-      'compression_level',
-      'client_id',
-      'auth_type',
-      'connection_type',
-      'username',
-      'sasl',
-      'partition',
-      'random',
-      'round_robin',
-      'hash',
-      'topics',
-      'headers',
-      'timeout',
-      'broker_timeout',
-      'required_acks',
-      'preset',
-      'secrets',
+    attributesToIncludeInAAD: new Set([
+      'service_token',
+      'shipper',
+      'allow_edit',
+      'broker_ack_reliability',
+      'broker_buffer_size',
+      'channel_buffer_size',
```

Comment:
I was not able to generate a ingest output object that contained any of these attributes. The two types that seemed to include encrypted properties (ssl, password), where Kafka and Logstash, and these did not contain any of the AAD properties. Any advice on how to better validate this object? Are these properties still relevant? Or are they a part of the AAD only by default of not adding them to the previous exclude list?

Reply:
`service_token` is set when creating a Remote Elasticsearch output with the Service Token stored as plain text. The broker and channel properties are optional properties of Kafka output, I think.

Reply:
I ended up just manually creating an object with all the included AAD fields.

Reply:
Unfortunately, even if the attributes are not relevant to encryption, we must keep them in AAD for compatibility. The new include list must contain all of the attributes that were not explicitly excluded from AAD for compatibility between versions - particularly for serverless upgrades, where the previous version of Kibana will be running alongside the new version of Kibana.